lib/ace/ext/papeeria/katex-demo.coffee

```diff
@@ -0,0 +1,31 @@
+define (require, exports, module) ->
+    require "ace/ext/jquery"
```

Comment:
So, again, just to be clear, should I not use the feature, which is braces around arguments are optional?